src/app/page/settings/settings.component.ts

```diff
@@ -36,6 +36,12 @@ export class SettingsComponent implements OnInit {
     });
   }
 
+  /** Test the Notification */
+  testNotify() {
+    return this.dataService.FireGET(API.TestNotification).subscribe(() => {
```

Comment:
Maybe this can listen to failure code (439) and display some message?
snackBar can be used for displaying message. 

But since this is api testing only feature, this is not a critical part.

Reply:
On re-seeing this, this is an empty subscribed function. @pulsejet requesting your comment on this (whether it is bad practise to have empty fn, or if it is okay).

Reply:
Won't break anything, but it's unnecessary. IIRC you can just do `observable.subscribe()` and that should still work.

But it _should not_ be empty. 
* The user must be given some form of indication that the API call actually succeed, otherwise they may keep repeatedly pressing the button. 
* Secondly, it's generally a good idea to disable the button while the call is in progress (e.g. disable the button)
* Also report failed calls to the user; don't confuse them

Reply:
> Maybe this can listen to failure code (439) and display some message?
> snackBar can be used for displaying message.
> 
> But since this is api testing only feature, this is not a critical part.

Just saw this one. I'll say at least display an `alert()` when the call succeeds/fails.